acsylla/_cython/session/session.pyx

```diff
@@ -74,7 +74,7 @@ cdef class Session:
             cass_future_free(cass_future)
 
 
-    async def execute(self, Statement statement):
+    async def execute(self, Statement statement, object return_all_pages=False):
```

Comment:
is it me or the `return_all_pages` is not really used?

Reply:
Unrelated to the PR but I missed changes like this [1] could be leaking memory since the CPP driver might be allocating memory that we are responsible to free up

[1] https://github.com/acsylla/acsylla/pull/53/files#diff-868c624f52f869a30fd8b9c40d43bdffa867165a7e292eb6c0ddb34ecd598d29R103

Reply:
My bad, I forgot clean code